pkg/cmd/generate-postgres-metadata-tables/main.go

```diff
@@ -100,3 +122,28 @@ func closeDB(conn *pgx.Conn) {
 		panic(err)
 	}
 }
+
+func getMappedType(dataType *string, dataTypeOid *uint32) error {
+	actualOid := oid.Oid(*dataTypeOid)
+	mappedOid, ok := unimplementedEquivalencies[actualOid]
+	if !ok {
+		// No mapped type
+		return nil
```

Comment:
in addition to the comment above, this case can be changed to `return dataType, dataTypeOID` (return the same input parameters)

Reply:
Done.